frontend/src/components/widgets/DataFrame/columns/NumberColumn.test.ts

```diff
@@ -193,12 +193,29 @@ describe("NumberColumn", () => {
     }
   )
 
-  it.each([[[]], ["foo"], [[1, 2]], ["123.124.123"], ["--123"], ["2,,2"]])(
-    "%p results in error cell",
-    (input: any) => {
-      const mockColumn = getNumberColumn(MOCK_FLOAT_ARROW_TYPE)
-      const cell = mockColumn.getCell(input)
-      expect(isErrorCell(cell)).toEqual(true)
-    }
-  )
+  it.each([
+    [[]],
+    ["foo"],
+    [[1, 2]],
+    ["123.124.123"],
+    ["--123"],
+    ["2,,2"],
+    ["12345678987654321"],
+  ])("%p results in error cell", (input: any) => {
+    const mockColumn = getNumberColumn(MOCK_FLOAT_ARROW_TYPE)
+    const cell = mockColumn.getCell(input)
+    expect(isErrorCell(cell)).toEqual(true)
+  })
+
+  it("shows an error cell if the numeric value is too large", () => {
+    const mockColumn = getNumberColumn(MOCK_INT_ARROW_TYPE)
+    const unsafeCell = mockColumn.getCell("1234567898765432123")
+    expect(isErrorCell(unsafeCell)).toEqual(true)
+    expect((unsafeCell as TextCell)?.data).toEqual(
+      "⚠️ 1234567898765432123\n\nThe value is larger than the maximum supported integer values in number columns (2^53).\n"
+    )
+
+    const safeCell = mockColumn.getCell("1234567898765432")
```

Comment:
Should this test be a separate test? 
I would expect this to be a separate test or the description to be longer. I prefer a separate test.

Reply:
Good point 👍  I will extract this as separate test